frappe/templates/discussions/button.html

```diff
@@ -1,7 +1,7 @@
 {% if frappe.session.user != "Guest" and
 (condition is not defined or (condition is defined and condition )) %}
-<span class="button is-secondary reply">
+<span class="btn btn-md btn-default reply">
 	{{ _(cta_title) }}
-	<svg class="icon icon-sm ml-1"><use href="#icon-add" style="stroke: var(--gray-700)"></use></svg>
+	<!-- <svg class="icon icon-sm ml-1"><use href="#icon-add" style="stroke: var(--gray-700)"></use></svg> -->
```

Comment:
Is it intentionally commented or did you miss removing it?

Reply:
It's intentional

Reply:
any reason?

Reply:
It was a part of the previous design. Not a part of the current design. There will be more design changes in the future. Hence keeping it commented till then. Once all the changes are implemented, I will remove all these pieces that won't be used.

Reply:
Just add a comment above that, a reviewer will know why it is commented.